src/game/sound_samples.c

```diff
@@ -10,14 +10,92 @@
 #include "system/nth_alloc.h"
 
 // TODO(#863): Sound_samples is not implemented
+// TODO: Volume control?
 
 struct Sound_samples
 {
     Lt *lt;
+    SDL_AudioDeviceID dev;
+    uint8_t **audio_buf_array;
+    uint32_t *audio_buf_size_array;
     size_t samples_count;
     int paused;
 };
 
+static 
+int init_buffer_and_device(Sound_samples *sound_samples, 
+                           const char *sample_files[]) 
+{
+    // TODO: Select audio specification from a menu
+    // TODO: Use a seperate callback function
```

Comment:
Could you please add more details to this TODO. It's unclear what needs to be done here.